docs/topics/signals.txt

```diff
@@ -189,6 +189,20 @@ In some circumstances, the code connecting receivers to signals may run
 multiple times. This can cause your receiver function to be registered more
 than once, and thus called multiple times for a single signal event.
 
+As Django has evolved, this issue has generally less risk of presenting itself.
+Before ``AppConfig.ready`` was introduced to Django, it was possible that
```

Comment:
Add which version this change was made? I think it helps add some more certainty to the time frame in which Django has 'evolved'.

Reply:
I'm not sure we should necessarily say the issue is "gone", since `AppConfig.ready` can be called multiple times during django's setup flow. I was mostly following through the instruction by the ticket (e.g. specifying that ready can be called multiple times, but at least that's a "standard" entrypoint that you can plan for). Maybe we should remove/reword this? Assuming the counter point is valid.

Reply:
Resolving for now as the wording has changed (and I've removed this idea/reference).